controllers/packagebundlecontroller_controller.go

```diff
@@ -140,14 +140,8 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 		}
 	}
 
-	bundles := &api.PackageBundleList{}
-	err = r.Client.List(ctx, bundles, &client.ListOptions{Namespace: api.PackageNamespace})
+	err = r.bundleManager.UpdateLatestBundle(ctx, latestBundle)
```

Comment:
I imagine your next move might be to combine this call with the `LatestBundle` call above?

Reply:
Yeh, definitely.